frontend/src/views/WheresMyGene/style.ts

```diff
@@ -44,6 +46,12 @@ export const Top = styled.div`
 
 export const SideBarWrapper = styled(SideBar)`
   max-height: calc(100vh - ${HEADER_HEIGHT_PX}px);
+
+  ${SideBarOpenButtonWrapper} {
```

Comment:
Added margin-bottom to the WMG sidebar toggle button to maintain consistency with production.